scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
Ping me on Slack when you guys have concluded ;) I'll go work on other platform stuff while I wait